framework/infrastructure/gcp/compute.py

```diff
@@ -652,11 +735,14 @@ def _execute(  # pylint: disable=arguments-differ
             request.headers[DEBUG_HEADER_KEY] = self.gfe_debug_header
             request.add_response_callback(self._log_debug_header)
         operation = request.execute(num_retries=self._GCP_API_RETRIES)
-        logger.debug("Operation %s", operation)
-        return self._wait(operation["name"], timeout_sec)
+        logger.info("Operation %s", operation)
```

Comment:
Please revert to `debug`.  You can enable debug logs with 

```
--logger_levels=__main__:DEBUG,framework.infrastructure.gcp:DEBUG
```

or more global

```
--logger_levels=__main__:DEBUG,framework:DEBUG
```